### PR TITLE
Add a new README

### LINKED ISSUE #632: [5pt] Make a new Readme
1. Move old readme's content to `./doc`
2. Make a new readme with just the quick start instructions and links to documentation

This issue is a part of #389

### PR DESCRIPTION
**Pull request by patiencedaur**
The old README has been moved to /doc. The new README is prettier.

Closes #632

### CODE REVIEW

**Comment by LeonidVas** [comment]
I am not very familiar with the task, but if we push this PR as is, then when entering to the project page in GH nothing will be displayed, which is very sad. Let's leave at least some information about the project with links to detailed documentation.

**Comment by patiencedaur** [comment]
@LeonidVas 
This is a draft pull request, that is, a work in progress. Of course, information will be added soon.

**Comment by NickVolynkin** [comment]
> I am not very familiar with the task, but if we push this PR as is, then when entering to the project page in GH nothing will be displayed, which is very sad. Let's leave at least some information about the project with links to detailed documentation.

This PR is marked as **draft** for a purpose. It's not done yet. Please come back when it's done and ready for review.

**Comment by mRrvz** [comment]
@NickVolynkin @patiencedaur I see that the PR is draft. Just a wish: please divide the commits into some logic groups (don't squash them into one as in the previous PR). It will be easier to review. Thanks! =)

**Comment by LeonidVas** [comment]
> @LeonidVas
> This is a draft pull request, that is, a work in progress. Of course, information will be added soon.

Ok.

File: README.rst [hunk 1]
```diff
@@ -1,1692 +1,132 @@
...
+*   `Checking instance status <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#checking-instance-status>`__
+*   `Displaying logs <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#displaying-logs>`__
+*   `Cleaning instance files <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#cleaning-instance-files>`__
+*   `Repairing a cluster <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#repairing-a-cluster>`__
+*   `Packaging your application <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#packaging-the-application>`__
 
-    #!/bin/sh
+    -   `Building a distribution <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#build-directory>`__
+    -   `Packing a TGZ <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#repair-commands>`__
+    -   `Packing an RPM or DEB distribution <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#rpm-and-deb>`__
```
Comment (Totktonada) [inline README.rst:127]: I like this structurization: packaging section and tgz, rpm, deb, docker subsection. However there are several comments around:

1. Packaging into a Docker image is missed in this list.
2. The ToC on the website does not follow this structure.
3. The link to the build step of the `pack` command looks useless in this list (why one would follow this link?). Maybe the sebsection is in the inappropriate place itself.

File: README.rst [hunk 2]
```diff
@@ -1,1692 +1,132 @@
...
 
-    # The main purpose of this script is to remove build artifacts from resulting package.
-    # It will run after `tarantoolctl rocks make` during application build.
+*   `Configuring an installed package <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#usage-example>`__
```
Comment (Totktonada) [inline README.rst:129]: I guess we should describe system a wide application configuration in its own section on the website too. However I see the logic here, despite the link itself is a bit confusing now (to a subsection or the 'RPM and Deb' section).

**Comment by Totktonada** [comment]
The new README.rst looks okay. I think the links in it should be synchronized with ToC on the website (or the ToC should be synced with the links... likely the both should be changed). However it is okay to leave it without the deep restructurization within this PR.

I would ask to squash the intermediate changes for the new README. So the final patchset may look like this:

* Move the old README to doc/
* Add the new README
* Add links to Tarantool Docs

Or, if it is hard to split intermediate commits:

* Move the old README to doc/
* Add the new README [assume it is with the links]

It seems, the latter should be as simple as squashing all commits except the first one.

I would like to push it to master in a good shape.

----

The recommendation about splitting different changes is correct and does not contract with my suggestion. The final result should contain atomic commits for readability of the git history, but it should not contain intermediate changes (fixups that you're doing during work and review) for exactly same reason.

A simple example how one may use git to present a patchset in a clear way:

1. Add a new functionality:

```diff
+function say_hello()
+    log.info('Hrrlo')
+end
```

2. Add another functionality:

```diff
+function say_bye()
+    log.info('Bye')
+end
```

The git history at this point (newest at top):

* Add say_bye()
* Add say_hello()

3. Observed a typo in the first commit:

```diff
 function say_hello()
-    log.info('Hrrlo')
+    log.info('Hello')
 end
```

The git history at this point:

* FIXUP: fix say_hello()
* Add say_bye()
* Add say_hello()

4. Squash the fixup into appropriate commit:

```sh
git rebase -i HEAD~3

# Edit it.
pick Add say_hello()
pick Add say_bye()
pick FIXUP: fix say_hello()

# To make look so.
pick Add say_hello()
f FIXUP: fix say_hello()
pick Add say_bye()
```

Now the history will be clean again without the intermediate change (the FIXUP commit):

* Add say_bye()
* Add say_hello()

**Comment by patiencedaur** [comment]
@Totktonada 
Thank you for your instructions! I squashed many small commits into one large as you requested. Kindly check if everything looks good enough so we can move on and restructure the README in another PR.

**Review by Totktonada — APPROVED** [review]
I'm ok. Please, consider given comments in the following PRs.

**Pull request merged**